examples/load-balanced/README.md

```diff
@@ -172,7 +179,137 @@ In this example, the PostgreSQL backups are written to the `/backups/postgresql`
+```
+
+Depending on what the status of the last backup is, Duply might run either a full or incremental backup. Incremental backup files contain just the diff of the state of the database dump after the last full backup. If you prefer to manually perform a full backup, run:
+
+```sh
+sudo su - postgres
+duply onadata full
+```
+
+Otherwise, Duply has been configured to perform a full backup weekly, and one incremental backup every day (at 1AM server time) after, for six days. Duply has been configured to maintain a maximum of four full backups. Backups are also kept for a maximum of thirty days. Update the `postgresql_backup_profiles` variable in [inventory/group_vars/postgresql/vars.yml](./inventory/group_vars/postgresql/vars.yml) if you'd want to change any of the above settings.
+
+To get the list of full and incremental backups, run:
+
+```sh
+sudo su - postgres
+duply onadata status
+```
+
+If the status command runs successfully, it should output statistics like below:
+
+```
+Found 0 secondary backup chains.
+
+Found primary backup chain with matching signature chain:
+-------------------------
+Chain start time: Thu Jul  2 12:53:39 2020
+Chain end time: Thu Jul  2 14:54:37 2020
+Number of contained backup sets: 4
+Total number of contained volumes: 4
+ Type of backup set:                            Time:      Num volumes:
+                Full         Thu Jul  2 12:53:39 2020                 1
+         Incremental         Thu Jul  2 14:43:42 2020                 1
+         Incremental         Thu Jul  2 14:54:32 2020                 1
+         Incremental         Thu Jul  2 14:54:37 2020                 1
+-------------------------
+No orphaned or incomplete backup sets found.
+--- Finished state OK at 14:54:47.327 - Runtime 00:00:00.164 ---
+```
+
+To restore the last backup (irregardless of whether it is a full or incremental backup), run:
+
+```sh
+sudo su - postgres
+duply onadata restore <path to file to place the database dump> 
+```
+
+Duply also allows you to restore a backup from a specific time. You can do this by adding the age of the backup to restore after the database dump path in the restore command above. Run `duply usage` for supported age formats.
+
+You can configure to receive notifications for when backups are either successful or fail. Do this using the `postgresql_backup_post_actions` variable. The variable accepts a list of dicts that configure either email or command actions. You can, for instance, add to it an item that makes Duply (the tool we use for backups) run the echo command if a backup fails. You can do the same with emails. For each of the items in the list you are allowed to leave out either the `on_error` or `on_success` dicts if you don't want an action done when a backup fails or is successful respectively. The example values below configure Duply to echo "Backup has failed" and send out when the backups either run successfully or fail:
```

Comment:
Is something missing from this sentence `The example values below configure Duply to echo "Backup has failed" and send out when the backups either run successfully or fail:`? the second portion"... and send out ..." seems odd to me. Can you revisit and state if this is clear for you?